### DISCUSSION TITLE
Question: decorator to check login?

### DISCUSSION THREAD

**Discussion by do9xe** [indico/flask-multipass#78] [Q&A] [answered]
Hi,
is there a decorator to check if a user is logged in before returning a view? I found `multipass.login_check()` but I`m not quite sure how to use it as documentation is a bit thin on it.
Any Help would be appreciated, I'd help improve the documentation, if I would know how to to this.

**Comment by ThiefMaster** [comment, marked as answer]
No, `login_check` is meant as a way for your app to tell multipass that the user is already logged in and thus should not go through the login process again.

And what you are asking for is outside the scope of Flask-Multipass. You may want to look into something like Flask-Login for this purpose. You'd then mark the user as logged-in there when you get the callback from Flask-Multipass.
